packages/graphql-yoga/src/server.ts

```diff
@@ -162,14 +151,20 @@ export type YogaServerOptions<
     TRootValue
   >
 
+  /**
+   * Envelop Plugins
+   * @see https://envelop.dev/plugins
+   */
+  plugins?: Array<
+    Plugin<TUserContext & TServerContext & YogaInitialContext> | Plugin | {}
+  >
+
   parserCache?: boolean | ParserCacheOptions
   validationCache?: boolean | ValidationCache
   fetchAPI?: FetchAPI
   multipart?: boolean
   id?: string
-} & Partial<
```

Comment:
I think we can prevent the users to leave `schema` and `plugins` undefined at the same time.

Reply:
IMHO it does not really make things safer as you can just define a plugin that uses no schema. 🤔  Thus IMHO we can just make both optional by default and don't introduce any type-complexity.